samples/nrf9160/modem_shell/src/ltelc/ltelc.c

```diff
@@ -0,0 +1,515 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <sys/types.h>
+
+#include <nrf9160.h>
+#include <hal/nrf_gpio.h>
+
+#include <shell/shell.h>
+#include <shell/shell_uart.h>
+
+#include <modem/at_cmd.h>
+#include <modem/modem_info.h>
+#include <modem/lte_lc.h>
+#include <modem/pdn.h>
+
+#include <nrf_socket.h>
+
+#include "ltelc_settings.h"
+#include "ltelc_shell.h"
+#include "ltelc_shell_print.h"
+
+#include "ltelc_shell_pdn.h"
+#include "ltelc_api.h"
+#include "ltelc.h"
+
+#if defined(CONFIG_MOSH_SMS)
+#include "sms.h"
+#endif
+
+#if defined(CONFIG_MODEM_INFO)
+#include <modem/at_cmd.h>
+#include <modem/at_cmd_parser.h>
+#include <modem/at_params.h>
+#endif
+
+static bool ltelc_subscribe_for_rsrp;
+
+static const struct shell *uart_shell;
+
+/* TODO: This is not used? */
+struct ltelc_pdn_socket_info_t {
```

Comment:
No need for `_t` suffix here (no typedef).

And it actually doesn't seem to be used like the comment says, so perhaps just remove it.

Reply:
Removed.